e2e/cypress/support/commands.js

```diff
@@ -86,8 +86,8 @@ Cypress.Commands.add('addFlight', (estString, actString, estChoxString, actChoxS
     "ActPax": 51,
     "TranPax": 0,
     "RunwayID": "05L",
+    "FlightID" : 100,
```

Comment:
Is there a reason FlightID is still here but with a different value?

Reply:
It's because the test class uses Chroma json which has a FlightID in it.